contribute/CONTRIBUTE-DOCS.rst

```diff
@@ -0,0 +1,294 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. contents:: :local:
+
+Overview
+========
+
+Contributing to documentation is one of the easiest and most welcome ways to improve Apache Airflow. Airflow documentation consists of the following components:
+
+- `Apache Airflow documentation <https://airflow.apache.org/docs/apache-airflow/stable/index.html>`__, which explains how to install and use Airflow.
+- `Provider documentation <https://airflow.apache.org/docs/apache-airflow-providers/index.html>`__, which explains how to connect Airflow to third-party provider packages.
+- `Helm chart documentation <https://github.com/apache/airflow/tree/master/docs/helm-chart`__, which explains how to run Airflow on Kubernetes using Helm
+- READMEs and contribution guidelines (including this page).
+- Inline docstrings, which describe functions within the source code itself.
+
+This guide provides guidance on working with docs, as well as best practices for making changes and submitting your work for review.
+
+What Should I Contribute?
+----------------------
+Do you want to contribute to Airflow docs, but you're not sure exactly where? Consider any of the following:
+
+- Typo fixes
+- Additional information about existing functionality
+- Documentation about a new feature
+- Images or screenshots that help provide users context
+- New tutorials
+
```

Comment:
For ideas, you can browse the open identified issues:

- `All GitHub Documentation Issues  <https://github.com/apache/airflow/issues?q=is%3Aopen+is%3Aissue+label%3Akind%3Adocumentation>`__

Reply:
Actually we can do a bit better https://github.com/apache/airflow/contribute - is a dedicated GitHub link for that label + it has `CONTRIBUTING.rst" link + nice invite explaining what those issues are.

Reply:
Ah no.. This is also for `kind:doc` - withdraw it.

Reply:
Good call to add this, @vikramkoka ! Made this change in my latest commit.